test/transform/CollectionUtilities.test.js

```diff
@@ -0,0 +1,102 @@
+import assert from 'assert'
+import domino from 'domino'
+import fixtureIO from '../utilities/FixtureIO'
+import pagelib from '../../build/wikimedia-page-library-transform'
+
+let document
+
+const CollectionUtilities = pagelib.CollectionUtilities
+
+describe('CollectionUtilities', () => {
+  beforeEach(() => {
+    document = fixtureIO.documentFromFixtureFile('CollectionUtilities.html')
+  })
+
+  describe('.collectPageIssuesText()', () => {
+    it('find text issues', () => {
+      const element = document.querySelector('div#content_block_0')
+      assert.deepEqual(CollectionUtilities.collectPageIssuesText(document, element), [
+        'This article includes a list of references, but its sources remain unclear because it has insufficient inline citations.  (January 2016)', // eslint-disable-line max-len
```

Comment:
There are different pros and cons and test styles but I'll just mention that these assertions would be simpler if the fixture was simplified (more unit-y) and less real world. No action is needed here but I wanted to share my thoughts.

Reply:
Good point. I was a bit lazy here :) These were mainly an easy check for me that my changes weren't breaking anything. I'll have a go at adding some more atomic tests...